deferral-2-0/Agile-Politico
# 164766247 Implement ability for profile update

**What does this PR do?**

Allow user to update their user profile on the system

**Description of Task to be completed?**


- [x] Write tests for the specified feature
- [x] Add the code for the specified feature

**How should this be manually tested?**

After cloning the repository to your local machine:
* Use Postman to add an account with your personal information
* Signin using the information
* Send a PATCH request to the '/users/<int:user_id>/edit' with some of your information different as is in the system
* Retrieve your profile information (alternatively, you can check the db records)
* You should be able to see that your information has been update to the newly provided one

**What are the relevant pivotal tracker stories?**

#164766247

**Relevant Screenshots**
![Screenshot from 2019-03-21 18-00-14](https://user-images.githubusercontent.com/38868860/54767714-d3e89900-4c0e-11e9-87b8-108de445ac95.png)
![Screenshot from 2019-03-21 18-00-05](https://user-images.githubusercontent.com/38868860/54767715-d4812f80-4c0e-11e9-8a0b-9cf4ccd7e4e0.png)
